internal/impl/envvars.go

```diff
@@ -136,5 +138,11 @@ func markEnvsAsSetByDevbox(envs ...map[string]string) {
 // as a proxy for this. This allows us to differentiate between global and
 // individual project shells.
 func (d *Devbox) IsEnvEnabled() bool {
-	return os.Getenv(d.ogPathKey()) != ""
+	env := map[string]string{}
+	for _, keyVal := range os.Environ() {
+		parts := strings.Split(keyVal, "=")
```

Comment:
Use strings.Cut, otherwise you'll lose the ends of variables that have an `=` in them. I think you could also export `envir.mapToPairs` and reuse it here.

Reply:
oh i like that. Done!